pkg/releaser/releaser.go

```diff
@@ -354,6 +347,31 @@ func (r *Releaser) CreateReleases() error {
 		if err := r.github.CreateRelease(context.TODO(), release); err != nil {
 			return errors.Wrapf(err, "error creating GitHub release %s", releaseName)
 		}
+
+		if r.config.PackagesWithIndex {
+			worktree, err := r.git.AddWorktree("", r.config.Remote+"/"+r.config.PagesBranch)
+			if err != nil {
+				return err
+			}
+			defer r.git.RemoveWorktree("", worktree) //nolint: errcheck
+
+			pkgTargetPath := filepath.Join(worktree, filepath.Base(p))
+			if err := copyFile(p, pkgTargetPath); err != nil {
+				return err
+			}
+
+			if err := r.git.Add(worktree, pkgTargetPath); err != nil {
+				return err
+			}
+
+			if err := r.git.Commit(worktree, fmt.Sprintf("Publishing chart package for %s", releaseName)); err != nil {
+				return err
+			}
+
+			if err := r.pushToPagesBranch(worktree); err != nil {
+				return err
```

Comment:
I guess this won't work if there are more than one packages to release as this can cause push failure because of conflict.

https://github.com/stecky/yet-another-chart-releaser/issues/3

We might move L352-L356 outside of this for loop.